lab1.3.css

```diff
@@ -0,0 +1,212 @@
+/* BASICS */
```

Comment:
is it actually needed to make full copy of css for each page? Can we use one css with custom selectors, or split it to smaller ones?